src/pykeen/evaluation/evaluator.py

```diff
@@ -548,6 +564,9 @@ def evaluate(
     # Prepare for result filtering
     if filtering_necessary or positive_masks_required:
         all_pos_triples = torch.cat([model.triples_factory.mapped_triples, mapped_triples], dim=0)
+        if additional_filter_triples is not None:
```

Comment:
Time for me to be the bad guy that I am...


```
   /$$                          /$$                                   /$$                                        
  | $$                         | $$                                  | $$                                        
 /$$$$$$    /$$$$$$   /$$$$$$$/$$$$$$   /$$$$$$$             /$$$$$$ | $$  /$$$$$$   /$$$$$$   /$$$$$$$  /$$$$$$ 
|_  $$_/   /$$__  $$ /$$_____/_  $$_/  /$$_____/            /$$__  $$| $$ /$$__  $$ |____  $$ /$$_____/ /$$__  $$
  | $$    | $$$$$$$$|  $$$$$$  | $$   |  $$$$$$            | $$  \ $$| $$| $$$$$$$$  /$$$$$$$|  $$$$$$ | $$$$$$$$
  | $$ /$$| $$_____/ \____  $$ | $$ /$$\____  $$           | $$  | $$| $$| $$_____/ /$$__  $$ \____  $$| $$_____/
  |  $$$$/|  $$$$$$$ /$$$$$$$/ |  $$$$//$$$$$$$/           | $$$$$$$/| $$|  $$$$$$$|  $$$$$$$ /$$$$$$$/|  $$$$$$$
   \___/   \_______/|_______/   \___/ |_______/            | $$____/ |__/ \_______/ \_______/|_______/  \_______/
                                                           | $$                                                  
                                                           | $$                                                  
                                                           |__/                                                  
```

Reply:
Is this just a random place for this announcement or did you have something in mind putting it here? 😊

Reply:
just a random place